docs/CONTRIBUTING.md

```diff
@@ -158,8 +158,14 @@ In order to build and run Enso you will need the following tools:
   need to configure the Developer Command Prompt for Microsoft Visual C++ for
   the x64 architecture.
 
-Managing multiple JVM installations can be a pain, so some of the team use
-[Jenv](http://www.jenv.be/): A useful tool for managing multiple JVMs.
+Managing multiple JVM installations can be a pain, so you can consider using
+helper tools for that. We recommend:
+
+- [Jenv](http://www.jenv.be/)
+- or [sdkman](https://sdkman.io/)
+
+**For users of M1 Mac**: installing GraalVM on M1 Mac is a bit complicated,
+please refer to a [dedicated documentation](./graalvm-m1-mac.md).
```

Comment:
The useful information here (or in the linked document) would be what specifically does not work or is "complicated" and how is this different from "usual" environment.

Reply:
Slightly reworded it